docs/modules/deploy/pages/deploying-in-kubernetes.adoc

```diff
@@ -50,3 +59,401 @@ Since all the Operators mentioned above are generated from Helm Charts, their fu
 == Enterprise Operator (Preview)
 
 Hazelcast Enterprise Operator is currently in the early alpha preview stage, but it will soon become the preferred way of installing Hazelcast in Kubernetes/OpenShift environments. For details, please check xref:deploy:deploying-with-operator.adoc[].
+
+== Discovering Members Automatically
+
+To make it easier to set up clusters in Kubernetes, Hazelcast allows members to discover each other automatically, using discovery modes.
+
+=== Discovery Modes
+
+The following table summarizes the differences between the discovery modes: **Kubernetes API** and **DNS Lookup**.
+
+[cols="1a,1a,1a"]
```

Comment:
I find that this table is somehow weirdly formatted.
![image](https://user-images.githubusercontent.com/2834997/126464348-08e38855-dedd-4a40-9be8-1d0a10ec554f.png)